save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/cvsscalculator/ScoreCalculator.kt

```diff
@@ -91,13 +92,18 @@ private fun calculate(baseMetrics: BaseMetrics): Float {
 @Suppress(
     "FLOAT_IN_ACCURATE_CALCULATIONS",
     "MagicNumber",
+    "VARIABLE_HAS_PREFIX",
 )
-// https://www.first.org/cvss/v3.1/specification-document#Appendix-A---Floating-Point-Rounding
```

Comment:
please leave all links in the comments

Reply:
I didn't remove the links. They were moved along with the method to another location